docs/socket.io-messages.md

```diff
@@ -1,10 +1,11 @@
 # List of message names used for socket.io communication between the API and client / CMS
 
-| Message name     | Description                       | Use                   | Data                                                      |
-| ---------------- | --------------------------------- | --------------------- | --------------------------------------------------------- |
-| connect          | connection established to API     | client & cms built-in | -                                                         |
-| clientDataReq    | Client data request               | client / cms -> API   | Object with client details such as current update version |
-| data             | Document transfer                 | API -> client / cms   | Object with Array of document objects and sync version    |
-| changeRequest    | Document changes                  | client / cms -> API   | Array of change requests                                  |
-| changeRequestAck | Acknowledgement of change request | API -> client / cms   | Acknowledgement of individual change request with status  |
-| accessMap        | Map with user specific access     | API -> client / cms   | Object                                                    |
+| Message name     | Description                                                                                           | Use                   | Data                                                      |
+| ---------------- | ----------------------------------------------------------------------------------------------------- | --------------------- | --------------------------------------------------------- |
+| connect          | connection established to API                                                                         | client & cms built-in | -                                                         |
+| clientDataReq    | Client data request                                                                                   | client / cms -> API   | Object with client details such as current update version |
+| data             | Document transfer                                                                                     | API -> client / cms   | Object with Array of document objects and sync version    |
+| changeRequest    | Document changes                                                                                      | client / cms -> API   | Array of change requests                                  |
+| changeRequestAck | Acknowledgement of change request                                                                     | API -> client / cms   | Acknowledgement of individual change request with status  |
+| accessMap        | Map with user specific access (depreciated)                                                           | API -> client / cms   | Object                                                    |
```

Comment:
It seems the code related to the `accessMap` event is removed, so does it serve a purpose to keep it here, even with the "deprecated" note?

Reply:
I still kept it for backwards compatibility (as the app and CMS are still on seperate duplications of the code handling the access map). As soon as it is implemented throughout it can be removed.